andela/boondocks-bn-backend
# #169257392 Set up Docker

#### What does this PR do?
Sets up Docker
#### Description of Task to be completed?

- [X] Added Dockerfile
- [X] Added .dockerignore
- [X] Added docker-compose.yml

#### How should this be manually tested?

1. Install Docker on your pc following instructions [here](https://docs.docker.com/docker-for-mac/install/)

2. Git clone this repo 

3. While on your project root: - run `docker-compose up --build`
                                                  - run `docker images` to make sure the image wa created
4. At successful creation of the image run `docker ps` get the container ID

5. Run docker exec it `container id` sh

6. Run `npm run migrate` 

7. Use Postman to test endpoint

#### Any background context you want to provide?
-N/A
#### What are the relevant pivotal tracker stories?
[#169257392](https://www.pivotaltracker.com/story/show/169257392)
#### Screenshot
None
#### Questions:
Can't one just push the image to the hub so that anyone can pull it then work with that?
